lib/ansible/galaxy/collection/__init__.py

```diff
@@ -199,17 +218,14 @@ def verify_local_collection(
             'in the following files:'.
             format(fqcn=to_text(local_collection.fqcn)),
         )
-        display.display(to_text(local_collection.fqcn))
-        display.vvv(to_text(local_collection.src))
         for content_change in modified_content:
             display.display('    %s' % content_change.filename)
-            display.vvv("    Expected: %s\n    Found: %s" % (content_change.expected, content_change.installed))
-        # FIXME: Why doesn't this raise a failed return code?
```

Comment:
@nitzmahone could you keep this `FIXME` note? This should probably be discussed separately.

Reply:
Yeah, I actually thought we *were* doing it when I removed it, then later realized we weren't- I almost added that once I realized... But it wouldn't be there anyway- it'd have to be up a layer for the `-r` case. But yeah, we'd need to mark something or return a structured result of some kind rather than just display output.

I'd actually prefer to just add the support, if we generally agree that a verify fail should be a nonzero return (+1 from me anyway)

Reply:
If we need `sys.exit(1)` one layer up, I'd prefer raising an exception to signal this. It could be caught and turned into an AnsibleError up the stack. During the latest refactoring it was quite clear that there's overuse of `AnsibleError` all over the place instead of doing the exit on the right level of abstraction but fixing this was out of scope...

Reply:
Yeah, `AnsibleError` is not the right thing in nearly any of the cases it's used in this tool... There are a lot of structural things that I'd love to change in here, but had to resist the urge :)

Reply:
resolved by https://github.com/ansible/ansible/pull/74051